appveyor.yml

```diff
@@ -4,6 +4,7 @@ image: Visual Studio 2015
 branches:
   only:
     - latestw_all
+    - hotfix

```

Comment:
are we going to maintain this hotfix branch? how different is it from latestw_all?